S23/mepripri/ToLowerCase.py

```diff
@@ -0,0 +1,49 @@
+import csv
+
+def splitLang(l):
```

Comment:
This function is great for getting the first part of the language input, but we should make `l` a little more descriptive in terms of the variable name.

Reply:
Sure, i will work on the variable name.